mods/doors/init.lua

```diff
@@ -700,6 +715,19 @@ doors.register_trapdoor("doors:trapdoor_steel", {
 	groups = {cracky = 1, level = 2, door = 1},
 })
 
+doors.register_trapdoor("doors:trapdoor_bar", {
+	description = "Steel Bar Trapdoor",
+	inventory_image = "doors_trapdoor_bar.png",
+	wield_image = "doors_trapdoor_bar.png",
+	tile_front = "doors_trapdoor_bar.png",
+	tile_side = "doors_trapdoor_bar_side.png",
+	protected = true,
```

Comment:
- Why is this protected while the door is not?
- I wonder if protection is good for these doors and trapdoors. One the one hand you can grab through the bars, on the other hand who cares about realism, these doors might loog very good for jails and there might be a lock making grabbing through useless.